### PR TITLE
[gh-pages][ZEPPELIN-996] Update first page of Zeppelin website

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### What is this PR for?
Apply same first page(`index.html`) to `gh-pages`.

### What type of PR is it?
Documentation

### What is the Jira issue?
[ZEPPELIN-996](https://issues.apache.org/jira/browse/ZEPPELIN-996)

### Screenshots (if appropriate)
screen shot 2016-06-15 at 10 52 48 pm
screen shot 2016-06-15 at 10 52 57 pm
screen shot 2016-06-15 at 10 53 07 pm

### Questions:
* Does the licenses files need update?No 
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -17,54 +17,51 @@ See the License for the specific language governing permissions and
...
-Currently Zeppelin supports many interpreters such as Scala(with Apache Spark), Python(with Apache Spark), SparkSQL, Hive, Markdown and Shell.
+## Multiple Language Backend
+[Apache Zeppelin interpreter](./docs/latest/manual/interpreters.html) concept allows any language/data-processing-backend to be plugged into Zeppelin.
+Currently Apache Zeppelin supports many interpreters such as Apache Spark, Python, JDBC, Markdown and Shell.
 
-<img class="img-responsive" src="assets/themes/zeppelin/img/screenshots/multiple_language_backend.png" />
+<img class="img-responsive" width="500px" style="margin:0 auto; padding: 26px;" src="./assets/themes/zeppelin/img/available_interpreters.png" />
 
-Adding new language-backend is really simple. Learn [how to write a zeppelin interpreter](./docs/latest/development/writingzeppelininterpreter.html).
+Adding new language-backend is really simple. Learn [how to create your own interpreter](./docs/latest/development/writingzeppelininterpreter.html#make-your-own-interpreter).
```
Comment (felixcheung) [inline index.md]: nit: `create your own` sounds a bit limiting - it's like it is for yourself only. How about 'create a new interpreter`?

**Comment by felixcheung** [comment]
LGTM

File: index.md (hunk 1 above)
Comment (AhyoungRyu) [inline index.md]: @felixcheung Yeah it's better. I've just addressed it :)

**Comment by bzz** [comment]
Looks great to me, thank you @AhyoungRyu @felixcheung !

Let's merge if there is no further discussion

**Pull request closed without merging**